src/main/java/kr/syeyoung/dungeonsguide/commands/CommandDgDebug.java

```diff
@@ -0,0 +1,100 @@
+package kr.syeyoung.dungeonsguide.commands;
+
+import kr.syeyoung.dungeonsguide.chat.ChatProcessor;
+import kr.syeyoung.dungeonsguide.utils.ScoreBoardUtils;
+import kr.syeyoung.dungeonsguide.utils.TitleRender;
+import net.minecraft.client.Minecraft;
+import net.minecraft.command.CommandBase;
+import net.minecraft.command.CommandException;
+import net.minecraft.command.ICommandSender;
+
+public class CommandDgDebug extends CommandBase {
+    @Override
+    public String getCommandName() {
+        return "dgdebug";
+    }
+
+    @Override
+    public String getCommandUsage(ICommandSender sender) {
+        return "dgdebug";
+    }
+
+    @Override
+    public void processCommand(ICommandSender sender, String[] args) throws CommandException {
+        if(args.length == 0) return;
+        String arg = args[0];
+
+
+
+        switch (arg){
+            case "scoreboard":
+                ScoreBoardUtils.forEachLine(l -> {
+                    ChatProcessor.INSTANCE.addToReciveChatQueue("LINE: " + l, false);
+                });
+
+
+                break;
+            case "scoreboardClean":
+                ScoreBoardUtils.forEachLineClean(l -> {
+                    ChatProcessor.INSTANCE.addToReciveChatQueue("LINE: " + l, false);
+                });
+
+
+                break;
+
+            case "title":
+                if(args.length == 2){
+                    System.out.println("Displayuing title:" + args[1]);
+                    TitleRender.displayTitle(args[1], "", 10, 40, 20);
+                }
+                break;
+            case "mockDungeonStart":
+                if(!Minecraft.getMinecraft().isSingleplayer()){
+                    ChatProcessor.INSTANCE.addToReciveChatQueue("This only works in singlepauer", false);
+                    return;
+                }
+
+                if(args.length == 2){
+                    int time = Integer.parseInt(args[1]);
+                    ChatProcessor.INSTANCE.addToReciveChatQueue("§r§aDungeon starts in "+time+" seconds.§r", false);
+                    return;
+                }
+
+
+                (new Thread(() -> {
```

Comment:
Make this into Feature class

Reply:
this is only for debugging the fancy dungeon countdown, no need for thread pools etc